src/components/PropEditors.js

```diff
@@ -167,7 +167,7 @@ class NumberEditor extends React.Component {
   }
 
   componentWillReceiveProps(props) {
-    this.setState({ value: String(props.value) });
```

Comment:
I wonder why I did that.  Does the underlying TextField do the stringification if a number is passed (as in, if `props.value` is a number?  (can test by editing a case that has a number in a numeric field)